### PR TITLE
Add support for NederHost DNS API

### PR DESCRIPTION
**Pull request by sebastiaanhoogeveen**
This adds support for the NederHost (https://www.nederhost.nl/) DNS API. Implementation is mostly based on dns_cf.sh with a few lines taken from dns_azure.sh.

### CODE REVIEW

**Comment by Neilpang** [comment]
please make sure you checked every requirements here: 
https://github.com/Neilpang/acme.sh/wiki/DNS-API-Dev-Guide

**Comment by sebastiaanhoogeveen** [comment]
Hi Neil,

The issue for API bug reports has been opened - the other items have been doublechecked and should be okay.

Thanks!

**Comment by Neilpang** [comment]
please resolve conflicts.

**Comment by sebastiaanhoogeveen** [comment]
Conflicts have been resolved.

File: dnsapi/dns_nederhost.sh [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+_nederhost_rest() {
+  m=$1
+  ep="$2"
+  data="$3"
+  _debug "$ep"
+
+  export _H1="Authorization: Bearer $NederHost_Key"
+  export _H2="Content-Type: application/json"
+
+  :>"$HTTP_HEADER"
```
Comment (Neilpang) [inline dnsapi/dns_nederhost.sh]: what is this line for ?

Comment (sebastiaanhoogeveen) [reply dnsapi/dns_nederhost.sh]: This is to make sure to clear any headers from a previous request; I need to parse the HTTP response code from the header to find the root zone. I took this line and the related grep/tail/cut/tr magic (on the line starting with _code=) from dns_azure.sh.

Comment (Neilpang) [reply dnsapi/dns_nederhost.sh]: please remove this line, and try again. I think it's not necessary.

Thanks.

Comment (sebastiaanhoogeveen) [reply dnsapi/dns_nederhost.sh]: You're right; tested it and works fine without the line. Thanks!

**Pull request merged**